CPUModel.Using.CSharp.Rec/Program.cs

```diff
@@ -43,6 +77,35 @@
 
 Console.ReadLine();
 
+class ForCommand
+{
+    private readonly ICommand[] _condition;
+    private readonly ICommand[] _increment;
+    private readonly ICommand[] _body;
+
+    public ForCommand(ICommand[] condition, ICommand[] body, ICommand[] increment)
```

Comment:
For loop can be a bit more complex:
for(int i = 0, j = 10; i < 10 && j > -5; i++, j--) {
 body
}

So there also should be a definition block that should be executed once, before loop.